### PR TITLE
CLOUD-92933 Convert MB to GB on OpenStack

### PR DESCRIPTION
**Pull request by topolyai5**

### CODE REVIEW

File: cloud-openstack/src/main/java/com/sequenceiq/cloudbreak/cloud/openstack/auth/OpenStackClient.java [hunk 1]
```diff
@@ -260,4 +263,19 @@ private void createAccessOrToken(AuthenticatedContext authenticatedContext) {
         return regions;
     }
 
+    public List<com.sequenceiq.cloudbreak.cloud.model.AvailabilityZone> getZones(OSClient osClient, String regionFromOpenStack) {
+        List<com.sequenceiq.cloudbreak.cloud.model.AvailabilityZone> availabilityZones = newArrayList();
```
Comment (mhmxs) [inline cloud-openstack/src/main/java/com/sequenceiq/cloudbreak/cloud/openstack/auth/OpenStackClient.java]: I suggest to use:
```
List<? extends AvailabilityZone> zonesFromOS = osClient.compute().zones().list();
        LOGGER.info("zones from openstack for {}: {}", regionFromOpenStack, zonesFromOS);
return zonesFromOS.stream().map(z -> availabilityZone(z.getZoneName())).collect(Collectors.toList());
```

File: cloud-openstack/src/main/java/com/sequenceiq/cloudbreak/cloud/openstack/common/OpenStackPlatformResources.java [hunk 2]
```diff
@@ -190,21 +185,33 @@ public CloudVmTypes virtualMachines(CloudCredential cloudCredential, Region regi
...
+        });
+
         CloudVmTypes cloudVmTypes = new CloudVmTypes(cloudVmResponses, defaultCloudVmResponses);
         LOGGER.info("openstack virtual machine types: {}", cloudVmTypes);
         return cloudVmTypes;
     }
 
+    private void convertVmSizeToGB(Iterable<VmType> types) {
+        types.forEach(t -> {
+            String key = "Memory";
```
Comment (mhmxs) [inline cloud-openstack/src/main/java/com/sequenceiq/cloudbreak/cloud/openstack/common/OpenStackPlatformResources.java]: `MEMORY_KEY` looks the same

File: cloud-openstack/src/main/java/com/sequenceiq/cloudbreak/cloud/openstack/common/OpenStackPlatformResources.java [hunk 3]
```diff
@@ -190,21 +185,33 @@ public CloudVmTypes virtualMachines(CloudCredential cloudCredential, Region regi
...
+            String memory = t.getMetaData().getProperties().get(key);
+            String formattedSize = ConversionUtil.convertToGB(memory);
```
Comment (mhmxs) [inline cloud-openstack/src/main/java/com/sequenceiq/cloudbreak/cloud/openstack/common/OpenStackPlatformResources.java]: What if is fail?

Comment (topolyai5) [reply cloud-openstack/src/main/java/com/sequenceiq/cloudbreak/cloud/openstack/common/OpenStackPlatformResources.java]: Memory key is exists in properties.

**Pull request merged**